packages/design-system-tokens/src/themes/cmsgov-component.ts

```diff
@@ -410,13 +410,11 @@ export const components: AnyTokenValues = {
     '__background-color--disabled':               t.color['gray-lighter'],
     '__border-width':                             '2px',
     '__border-width--error':                      '3px',
-    '__border-width--success':                    '3px',
     '__border-color':                             t.color['base'],
     '__border-color--disabled':                   t.color['gray-light'],
     '__border-color--error':                      t.color['error'],
     '__border-color--error--inverse':             t.color['error-light'],
     '__border-color--inverse':                    t.color['black'],
-    '__border-color--success':                    t.color['success-light'],
```

Comment:
do we still need the `success-light` color token?

Reply:
nevermind! just remembered we use this for success alarts and badges and stuff

Reply:
We're still using it in other places, and we made a push a while back to have a full gradient of each of our color families even if we don't necessarily use them all (like `-lightest` to `-darkest`)